newjersey/navigator.business.nj.gov
# feat: [#183913408] create header styling to support contextual links …

…in the profiles



## Description

We wanted to make the Headers in profile support contextual links



### Ticket



[183913408](https://www.pivotaltracker.com/story/show/#183913408)

### Approach



As discussed during a standup we wanted to render the contextual link as a separate field instead of a markdown field for reusability of the header text across the application.

I also did some of the copy but not all of it as some of it doesn't have contextual link files ready yet (Specifically for Dakota DBA). 

I also made some fields, required optional as the new designs don't include these fields and they have been required up to this point

### Steps to Test



Go to profile as the various users and see that contextual links are rendered and that the tabs accurately represent the design in the pivotal ticket / figma (except for Dakota)

Also make sure you have different onboarding styling for the legal structure question (the onboarding one should have a description while the other should just be a title and the profile one should just be a contextual link)

### Notes





## Code author checklist
